be/src/util/bthreads/util.h

```diff
@@ -51,7 +51,7 @@ StatusOr<bthread_t> start_bthread(std::function<void()> func) {
 // Starts a new bthread that runs the specified function, then waits for the new bthread
 // to complete before returning to the caller.
 // Note: The function provided must not throw any exceptions.
-Status start_bthread_and_join(std::function<void()> func) {
+inline Status start_bthread_and_join(std::function<void()> func) {
```

Comment:
My question is why you don't move the implementation into the cpp source file?

Reply:
That's also an option, I'm just choosing an option with less modification.